util-taglib/src/com/liferay/taglib/ui/SuccessTag.java

```diff
@@ -107,6 +111,8 @@ public boolean isTranslateMessage() {
 	public int processEndTag() throws Exception {
 		String message = _message;
 
+		String messageType = _messageType;
```

Comment:
No need for this variable here. It could be directly referenced in https://github.com/liferay-frontend/liferay-portal/pull/173/files#diff-b2cb7b0ecbaa813d2951439376b509b4R141

Reply:
I just followed the pattern of message but I noticed now message variable was being changed during the method.

Updating using the class-defined-instance directly.